pkg/sql/vars.go

```diff
@@ -1308,6 +1308,15 @@ var varGen = map[string]sessionVar{
 		},
 	},
 
+	// See https://www.postgresql.org/docs/9.4/runtime-config-connection.html
+	`ssl`: {
+		Hidden: true,
+		Get: func(evalCtx *extendedEvalContext, _ *kv.Txn) (string, error) {
+			insecure := evalCtx.ExecCfg.RPCContext.Config.Insecure || evalCtx.ExecCfg.RPCContext.Config.AcceptSQLWithoutTLS
+			return formatBoolAsPostgresSetting(!insecure), nil
+		},
```

Comment:
On [pg15's doc](https://www.postgresql.org/docs/15/runtime-config-connection.html) it says `This parameter can only be set ... on the server command line`. Do we want to enable the set method here as well?

Reply:
no, i don't think so. since we don't support the same command line arguments or configuration files that PG has.

the analogous command line argument for CRDB is `cockroach start --insecure`.